controllers/rideRequestController.js

```
@@ -0,0 +1,132 @@
+          return responseSuccessWithMessage(
+            res,
+            updatedRideRequest,
+            RIDE_REQUEST_UPDATED
+          );
+        } else {
+          const requestExist = await tripRequest.findOne({
+            where: filterCondition,
+          });
+
+          if (!requestExist) {
+            filterCondition = {
```

Biome (2.1.2)

[error] Can't assign filterCondition because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)